backend/crawlers/ranking.js

```diff
@@ -604,9 +607,13 @@ const crawler = async (delayedStart) => {
     logger.debug(loggerOptions, `Last era in DB is ${lastEraInDb}`);
 
     // thousand validators program data
-    logger.debug(loggerOptions, 'Fetching thousand validator program validators ...');
-    const thousandValidators = await getThousandValidators();
-    logger.debug(loggerOptions, `Got info of ${thousandValidators.length} validators from Thousand Validators program API`);
+    // https://wiki.polkadot.network/docs/thousand-validators
+    let thousandValidators;
```

Comment:
Let's use an empty array here. So you don't need to add extra checks in the code below

Reply:
Done